tgstation.dme

```diff
@@ -2197,7 +2197,6 @@
 #include "code\modules\atmospherics\machinery\components\trinary_devices\mixer.dm"
 #include "code\modules\atmospherics\machinery\components\trinary_devices\trinary_devices.dm"
 #include "code\modules\atmospherics\machinery\components\unary_devices\bluespace_sender.dm"
-#include "code\modules\atmospherics\machinery\components\unary_devices\cryo.dm"
```

Comment:
Extremely uncomfortable with this

Reply:
Yea, this shouldn't be ticked off, but rather just commented out entirely.

Reply:
That's also not great, we should just be making the adjustments we need to make to the original cryogenics code.

Reply:
I'm fine with just changing the layer value in the original cryo code, I did this based on it saying if it can be modularised do it.